presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -48,8 +49,16 @@ Nessie catalog
 ^^^^^^^^^^^^^^
 
 In order to use a Nessie catalog, ensure to configure the catalog type with
```

Comment:
Maybe this line could be rephrased as shorter without changing the meaning? Consider if "To use a Nessie catalog, configure the catalog type as" still means what you want it to mean here.